calculator.rb

```diff
@@ -1,3 +1,8 @@
+#Constants defined
```

Comment:
I like that you separated this file with comments as it makes the program easier to read